schemas/oic.links.properties.core-schema.json

```diff
@@ -84,6 +84,26 @@
                   }
               }
           }
+      },
+      "tag-pos-desc": {
+        "description": "Descriptive relative position of the Resource",
+        "$ref": "oic.types-schema.json#/definitions/pos-descriptions"
+      },
+      "tag-rel-pos": {
```

Comment:
In the CR, the Link Parameter is called "tag-pos-rel" (i.e. flip the "rel" and "pos".

Reply:
Fixed